rancid-tomatillos/src/apiCalls.js

```diff
@@ -0,0 +1,19 @@
+export const fetchUser = (options) => {
```

Comment:
Nice work starting to get the fetches pulled out into a seperate file to make them reusable and easier to test!